tests/ssp/tests/01-run.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+
+def testfunc(child):
+    child.expect_exact('calling stack corruption function')
+    child.expect('ssp: stack smashing detected')
```

Comment:
on my system (native), I have the following string:
```
*** stack smashing detected ***: <unknown> terminated
make: *** [/tmp/RIOT/tests/ssp/../../Makefile.include:387: term] Aborted (core dumped)
```

So I would propose that you change this line to:
```
child.expect('.*stack smashing detected.*')
```
and remove the next line (\*\*\* halted) as it is not displayed in native (for me).